README.md

```diff
@@ -1,3 +1,287 @@
+## Workflows/Actions
+**You can also find the implementation of the workflows in the [`skill-template-repo`](https://github.com/OpenVoiceOS/skill-template-repo)**
+
+### Publish Alpha Release (example: `ovos-core`)
+```yaml
+name: Publish Alpha Build ...aX
+on:
+  push:
+    branches:
+      - dev
+    paths-ignore:
+      - 'ovos_core/version.py'
+      - 'test/**'
+      - 'examples/**'
+      - '.github/**'
+      - '.gitignore'
+      - 'LICENSE'
+      - 'CHANGELOG.md'
+      - 'MANIFEST.in'
+      - 'readme.md'
```

Comment:
should  it be README? or better, have both uppercase and lowercase?

Reply:
I've seen both `readme.md` and `README.md` in different ovos repos

Reply:
I have been renaming to upper case whenever i notice it for consistency, but we should consider both valid by default

Reply:
maybe i'll add a comment that those have to be adjusted :P